src/components/ManageUsers/SearchUsers.js

```diff
@@ -60,8 +60,14 @@ class SearchUsers extends Component {
               ADD NEW
             </button>
           </div>
-          <Modal isOpen={this.state.toggleAddModal}>
-            <AddUser userid={this.props.userid} />
+          <Modal
+            isOpen={this.state.toggleAddModal}
+            toggle={this.toggleAddModal}
+          >
```

Comment:
why are these the same... that seems confusing
one should be a function and one should be a boolean.
Im sure thats what they are but it looks like they are both functions just from the name

Reply:
I think line 64 is accessing a boolean on state called `toggleAddModal` and the other is the component method `toggleAddModal`, but I agree, the wording is confusing